pkg/virt-controller/watch/vmi.go

```diff
@@ -603,6 +612,46 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 	return nil
 }
 
+// Checks if PVC used in VolumeSource is owned by a DataVolume.
+// If so, the DV should be used instead and this is an error
+func (c *VMIController) handlePVCMisuseLALA(vmi *virtv1.VirtualMachineInstance) error {
```

Comment:
you're probably still working on this change and I'm reviewing while you're in the middle of  things. just a reminder to remove this.

Reply:
Oh... :man_facepalming: :smile: